numba/cuda/compiler.py

```diff
@@ -989,23 +977,6 @@ def specialized(self):
         """
         return len(self.sigs) == 1 and not self._can_compile
 
-    @property
-    def definition(self):
-        warn('Use overloads instead of definition',
-             category=NumbaDeprecationWarning)
-        # There is a single definition only when the dispatcher has been
-        # specialized.
-        if not self.specialized:
-            raise ValueError("Dispatcher needs to be specialized to get the "
-                             "single definition")
-        return next(iter(self.overloads.values()))
-
-    @property
-    def definitions(self):
-        warn('Use overloads instead of definitions',
-             category=NumbaDeprecationWarning)
-        return self.overloads
-
```

Comment:
These aren't mentioned in the deprecation schedule docs?

Reply:
It turns out that they're not, and that was an oversight - they'll have been warning in any code that uses them since 0.53 though.

Reply:
Ah yes, they have been.